repeat.mjs

```diff
@@ -0,0 +1,5 @@
+import { execSync } from "child_process";
```

Comment:
this file was only used for local debugging (https://github.com/cloudflare/workers-sdk/commit/6efb5c91a69592112eb2969c28dd0ee092044a84) and no longer needed no?